Source/player.cpp

```diff
@@ -3244,7 +3245,7 @@ void CheckNewPath(int pnum)
 			i = plr[pnum].destParam1;
 			x = abs(plr[pnum]._px - monster[i]._mfutx);
 			y = abs(plr[pnum]._py - monster[i]._mfuty);
-			if (x <= 1 && y <= 1) {
+			if (x <= 1 && y <= 1 || holdPosition) {

```

Comment:
Should parentheses be added for clarity?
```suggestion
			if ((x <= 1 && y <= 1) || holdPosition) {

```

Reply:
Definitely. Added.